dtApi/src/app/login/services/auth.service.ts

```diff
@@ -0,0 +1,41 @@
+import { HttpClient } from '@angular/common/http'
+import { Injectable } from '@angular/core'
+import { Observable } from 'rxjs'
+import { map, tap } from 'rxjs/operators'
+import { Router } from '@angular/router'
+
+@Injectable({
+    providedIn: 'root',
+})
+export class AuthService {
+    constructor(private http: HttpClient, private router: Router) {}
+    loginRequest(userName: string, password: string): Observable<any> {
+        const url = 'https://dtapi.if.ua/api/login'
```

Comment:
move host url to app environments and import it from there

Reply:
So, at environment folder you have two files 
- environment.ts
- environment.prod.ts
please put BASE URL https://dtapi.if.ua/api into these files